docs/content/users/quickstart.md

```diff
@@ -176,18 +176,18 @@ Environment variables will be automatically added to your `.env` file to simplif
     ddev config --project-type=craftcms --docroot=web
 
     # Boot the project and install the starter project:
-    ddev start
     ddev composer create-project --no-scripts craftcms/craft
-    ddev craft install/craft \
+
+    # Install Craft CMS:
+    ddev craft install \
         --username=admin \
         --password=Password123 \
         --email=admin@example.com \
-        --site-name='$DDEV_PROJECT' \
         --language=en \
-        --site-url='$DDEV_PRIMARY_URL'
```

Comment:
Since this gets persisted to yaml, we don't want the `DDEV_` version.

Reply:
We can't assume people using this quickstart have the latest DDEV, so it might not work for them.
(BTW, we use `$DDEV_PRIMARY_URL` for many quickstarts.)

Reply:
@stasadev not the end of the world if we don't change it – but in Craft's case, using it gets very awkward: The unparsed value `$DDEV_PRIMARY_URL` ends up getting built into a yaml config.

While it is normal to reference an env var in the yaml like this (especially for site url), you would never want the var to be called `DDEV_` as that would indicate that you would name it that way in another/remote environment.